README.md

```diff
@@ -2,3 +2,18 @@
 
 This repository holds coala Enhancement Proposals. You can read more about them
 in the `cEP-0000.md` document.
+
+## Index
+ |       Filename        |       Title       |                    Description                      |
+|-----------------------|-------------------|-----------------------------------------------------|
+|[cEP-0000](cEP-0000.md)|coala Enhancement Proposals|This cEP describes the official roles members of the coala community can hold and how they can vote on different levels of decisions.|
```

Comment:
please add a space to separate the text and the column markers.

Also try to align the column markers so that they are vertically aligned.

Reply:
okay, done. Thank you